0.1/index.bs

```diff
@@ -2,7 +2,7 @@
 Title: Next-generation file formats (NGFF)
 Shortname: ome-ngff
 Level: 1
-Status: w3c/ED
+Status: w3c/CG-FINAL
```

Comment:
The list of permissible values is available under https://tabatkins.github.io/bikeshed/#metadata

```
ED: "Editor’s Draft"
WD: "W3C Working Draft"
FPWD: "W3C First Public Working Draft"
LCWD: "W3C Last Call Working Draft"
CR: "W3C Candidate Recommendation"
PR: "W3C Proposed Recommendation"
REC: "W3C Recommendation"
PER: "W3C Proposed Edited Recommendation"
WG-NOTE: "W3C Working Group Note"
IG-NOTE: "W3C Interest Group Note"
MO: "W3C Member-only Draft"
UD: "Unofficial Proposal Draft"
CG-DRAFT: "Draft Community Group Report"
CG-FINAL: "Final Community Group Report"
```

"REC", "CR", and "PR" all require other metadata (list of implementations) that we _may_ want to provide, but I don't think we're there yet. This renders as:

![Screen Shot 2020-11-25 at 17 03 23](https://user-images.githubusercontent.com/88113/100252430-24fe7380-2f40-11eb-984c-2a9d0de2f5a0.png)